RundeckAutomation/Functions/New-RundeckSession.ps1

```diff
@@ -0,0 +1,64 @@
+function New-RundeckSession

```

Comment:
If you're going to keep the global, internal session, rename this function to `Set-RundeckSession`. The `New` verb usually creates a thing and returns it.